page_creation.rst

```diff
@@ -273,7 +285,7 @@ project:
 ``src/``
     All your PHP code lives here.
 
-99% of the time, you'll be working in ``src/`` (PHP files) or ``config/`` (everything
+Most of the time, you'll be working in ``src/`` (PHP files) or ``config/`` (everything
```

Comment:
Maybe we should remove this phrase entirely? You now have lots of dirs to work on (templates, translations, assets, ...) It's not true that most of the time you are in src/ and config/